workflows/synstage.nf

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env nextflow
+
+// Ensure DSL2
+nextflow.enable.dsl = 2
+
+/*
+========================================================================================
+    SETUP PARAMS
+========================================================================================
+*/
+
+input_file = file(params.input, checkIfExists: true)
+workdir = "${workDir.parent}/${workDir.name}"
+params.outdir = "${workDir.scheme}://${workdir}/synstage/"
+params.outdir_clean = params.outdir.replaceAll('/$', '')
+params.input_parent_dir = input_file.parent
+// Parse Synapse URIs from input file
+params.synapse_uris = (input_file.text =~ 'syn://(syn[0-9]+)').findAll()    
+// Parse SBG URIs from input file
+params.sbg_uris = (input_file.text =~ 'sbg://([a-zA-Z0-9]+)').findAll()
+
+/*
+========================================================================================
+    IMPORT MODULES
+========================================================================================
+*/
+
+include { SYNAPSE_GET } from '../modules/synapse_get.nf'
+include { SEVENBRIDGES_GET } from '../modules/sevenbridges_get.nf'
+include { UPDATE_INPUT } from '../modules/update_input.nf'
+
+workflow SYNSTAGE {
+
+    // Warning if input file lacks URIs
+    if (params.synapse_uris.size() == 0 && params.sbg_uris.size() == 0) {
+    message = "The input file (${params.input}) does not contain any Synapse " +
```

Comment:
For formatting does this allow you to indent by 1 from the `if` statement above?